projects/netgrif-components-core/src/lib/forms/email-submission/abstract-email-submission-form.component.ts

```diff
@@ -1,12 +1,14 @@
 import {EventEmitter, Input, OnDestroy, Output} from '@angular/core';
 import {FormBuilder, FormGroup, Validators} from '@angular/forms';
 import {FormSubmitEvent, HasForm} from '../has-form';
+import {Observable} from 'rxjs';
 
 export abstract class AbstractEmailSubmissionFormComponent implements HasForm, OnDestroy {
 
     public rootFormGroup: FormGroup;
 
     @Input() public displayLegalNotice = true;
+    @Input() public loading = new Observable<boolean>();
```

Comment:
It should be the type of `LoadingEmtter`.

Reply:
@minop because the event (endpoint calling and receiving response) is happening in it's parent view. But yes, it can be done in other way, e.g. passing the LoadingEmitter in the form submissionevent.

@mladoniczky corrected